coral/src/services/api.ts

```diff
@@ -64,7 +64,7 @@ const API_PATHS = {
   updateProfile: "/updateProfile",
   updatePermissions: "/updatePermissions",
   updateKwCustomProperty: "/updateKwCustomProperty",
-  udpateTenant: "/udpateTenant",
+  updateTenant: "/updateTenant",
```

Comment:
I had to copy the word on the left and search for it, my 🧠  did NOT see the difference. Nice catch!

Reply:
Murali caught it, I was just confused XD

Reply:
... took me a few seconds AGAIN :D